newrelic-security-agent/src/main/java/com/newrelic/agent/security/intcodeagent/models/javaagent/ExitEventBean.java

```diff
@@ -6,7 +6,7 @@
 public class ExitEventBean extends AgentBasicInfo {
     private String executionId;
     private String caseType;
-    private String k2RequestIdentifier;
+    private String nrRequestIdentifier;
```

Comment:
please change this to csec request identifier

Reply:
Changes provided in [commit](https://github.com/newrelic/csec-java-agent/pull/215/commits/d03a43fd1e12f22085a89121d90b7a1ddaed85b3)